src/args/lint-nix/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+# shellcheck shell=bash
+
+function main {
+  export LANG=C.UTF-8
+  source __argTargets__/template local targets
+
+  info Linting Nix code \
+    && for target in "${targets[@]}"; do
+      info Linting "${target}" \
+        && if ! statix check "${args[@]}" "${target}"; then
```

Comment:
Does it use the strictest config by default?

Reply:
Yes, [statix configuration](https://github.com/nerdypepper/statix#configuration) is only to disable rules, it enables every rule by default.